app/src/main/java/com/keylesspalace/tusky/components/preference/PreferencesFragment.kt

```diff
@@ -280,23 +280,24 @@ class PreferencesFragment : PreferenceFragmentCompat(), Injectable {
     }
 
     private fun updateHttpProxySummary() {
-        val sharedPreferences = preferenceManager.sharedPreferences
-        val httpProxyEnabled = sharedPreferences.getBoolean(PrefKeys.HTTP_PROXY_ENABLED, false)
-        val httpServer = sharedPreferences.getNonNullString(PrefKeys.HTTP_PROXY_SERVER, "")
+        preferenceManager.sharedPreferences?.let { sharedPreferences ->
```

Comment:
```suggestion
        val sharedPreferences = preferenceManager.sharedPreferences ?: return
```

(don't apply it, just an example)